src/v2/view-builder/views/TerminalView.js

```diff
@@ -159,8 +159,9 @@ const Footer = BaseFooter.extend({
     if (this.options.appState.containsMessageWithI18nKey(DEVICE_CODE_ERROR_KEYS)) {
       return getReloadPageButtonLink();
     }
-
-    if (!this.options.appState.containsMessageWithI18nKey(NO_BACKTOSIGNIN_LINK_VIEWS)) {
+    // If cancel object exists idx response then view would take care of rendering back to sign in link
```

Comment:
Correct if I'm wrong but I think the comment meant to say: If cancel object does not exist?

Reply:
Here there are two flavors of `Back to sign in`.. 

line 165 adds a link `if the view expect a link but there is no cancel object`

if the `cancel` object exists then do not execute line `165` and let View render it so that we can avoid two duplicate links.